### ISSUE TITLE
CRITICAL - Could not interpret output from ping command

### ISSUE THREAD

**Issue by babvin** [manios/docker-nagios#30]
Getting an error for localhost (container) when trying to start the Nagios. 
![2021-07-09 21_14_35-Nagios_ 192 168 15 3](https://user-images.githubusercontent.com/30034691/125104912-83f1b100-e0fb-11eb-88d8-332ce221bac0.png)
![Uploading 2021-07-09 21_14_35-Nagios_ 192.168.15.3.png…]()

**Comment by manios** [comment]
Hi @babvin ! 

Can you please provide me with the following information? 

**STEP 1**: Please run `sh` in your container:

```bash
docker exec -it nagios  apk --print-arch
```

and provide me the output of the command? For example the output in my PC is:

```
x86_64
```

**STEP 2**: Also can you tell me where do you run the container? In a PC, Raspberry 1,2,3,4?

Thanks!

**Comment by babvin** [comment]
Hi Manios

I’m running it on Ubuntu 20.04 - Aarch64 / Raspberry Pi 4

Best regards
Vinay

**Comment by manios** [comment]
Hi @babvin !

Thanks for the info! I still need the following information:

STEP 1: Please execute the following to your running container:

```bash
docker exec -it nagios  apk --print-arch
```
and provide me the output of the command? For example the output in my PC is:
```
x86_64
```

Thanks!
Christos

**Comment by manios** [comment]
Hi @babvin !

I have released a new `latest` tag based on `alpine:3.14` with the appropriate changes. Can you please give it a try

```shell
docker pull manios/nagios:latest
# or
docker pull manios/nagios:4.4.6
```

and provide me with feedback ? I am closing the issue. If you find any discrepancy please post a comment here and we will take a look.

Thanks,
Christos
